src/app/components/AlQuran.tsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
we are returning the Functional component here right? Or if it is an icon why not renaming the file and moving it to somewhere else. Because I believe if they aren't the components they should not be kept in here

Reply:
Same goes with `components/Hadith.tsx`